includes/class-sensei-grading.php

```diff
@@ -258,10 +295,29 @@ public function grading_user_quiz_view() {
 		$sensei_grading_user_profile = new Sensei_Grading_User_Quiz( $user_id, $quiz_id );
 
 		// Wrappers
+		/**
+		 * Fires before the container of the Grading page.
+		 *
+		 * @hook grading_before_container
+		 */
 		do_action( 'grading_before_container' );
+
+		/**
+		 * Fires before the container of the Grading page.
+		 *
+		 * @hook grading_before_container
```

Comment:
```suggestion
		 * @hook grading_wrapper_container
```

Is the description up-to-date?

Reply:
> Is the description up-to-date?

I think so, but it is pretty general. If you have a better one, we can use it.

Reply:
I added a clarification for all wrapper actions: https://github.com/Automattic/sensei/pull/7638/commits/7a0acbd7bd94cadd7d30bfa9fa5dbd31730869e9